doc/sphinx/addendum/miscellaneous-extensions.rst

```diff
@@ -7,20 +7,24 @@ of program refinements. To use the Derive extension it must first be
 required with ``Require Coq.derive.Derive``. When the extension is loaded,
 it provides the following command:
 
-.. cmd:: Derive @ident__1 SuchThat @one_term As @ident__2
-
-   :n:`@ident__1` can appear in :n:`@one_term`. This command opens a new proof
-   presenting the user with a goal for :n:`@one_term` in which the name :n:`@ident__1` is
-   bound to an existential variable :g:`?x` (formally, there are other goals
+.. cmd:: Derive @open_binders in @type as @ident
+         Derive @open_binders SuchThat @type As @ident
+
+   where :n:`@open_binders` declares a list of declarations of the form
+   :g:`@ident__i : @type__i` which can appear in :n:`@type`. This
+   command opens a new proof presenting the user with a goal for
+   :n:`@type` in which each name :n:`@ident__i` is bound to an
+   existential variable :g:`?x__i` (formally, there are other goals
    standing for the existential variables but they are shelved, as
    described in :tacn:`shelve`).
 
-   When the proof ends two :term:`constants <constant>` are defined:
+   When the proof ends as many :term:`constants <constant>` as the
+   number of :g:`@ident__i` plus one are defined:
```

Comment:
```suggestion   
   When the proof is complete, Coq defines :term:`constants <constant>` for each :n:`@ident__i` and for :n:`@ident`:
```

Would you clarify what you mean by defining a constant?  Is it an identifier tied to the proof or is visible outside the proof, such as for the `Inductive` command?